RELEASE-NOTES.txt

```diff
@@ -1,3 +1,9 @@
+1.3.6
+=====
+* Fixed issue with resolving relative symlink paths #1009
+* Fixed issue with WP.org API zod schema #1003
```

Comment:
```suggestion
```
This is entirely invisible to users, so I'd argue that we can skip it.

Reply:
Thanks, removed.